pages/1.11/release-notes/1.11.0-rc1/index.md

```diff
@@ -86,6 +86,7 @@ To download DC/OS Enterprise, contact: [Mesosphere Support](https://support.meso
 
 # <a name="known-issues"></a>Known Issues and Limitations
 - Upgrades from 1.10 to 1.11 are _not supported_ in 1.11 Release Candidate 1.
+- dcos-secrets service is unavailabe during upgrade and it's consequences (DCOS-19047).
```

Comment:
Format for the previous release notes is:
- \<Jira number\> - short description

What does "it's consequences" mean here?